src/main/resources/application.properties

```diff
@@ -159,6 +159,12 @@ config.import.configsMaxCount=${IMPORT_CONFIGS_MAX_COUNT:64}
 features.flags.addonsSupported=${ADDONS_SUPPORTED:false}
 features.flags.assistantsSupported=${ASSISTANTS_SUPPORTED:false}
 
+deployment.cache.expiration.interval=${DEPLOYMENT_CACHE_EXPIRATION_INTERVAL_MS:300000}
+deployment.client.url=${DEPLOYMENT_CLIENT_URL}
+
+endpoints.refresh.enabled=${ENABLE_ENDPOINTS_REFRESH:false}
```

Comment:
i'd group them by name (use the same prefix part). all new props are logically related to the same deployment_manager component

Reply:
changed